src/app/modules/dogs/store/actions.ts

```diff
@@ -1,16 +1,16 @@
 import { Dog } from "@app/shared/models/dog";
 import { Action } from "@ngrx/store";
 
-export class SetDogs implements Action {
-  public readonly type = '[dogs] setDogs';
+export class SetAllDogs implements Action {
```

Comment:
Consider renaming to SetAllDogsAction to avoid confusion (applies to all actions)